packages/inventory-compliance/src/ComplianceRemediationButton/ComplianceRemediationButton.js

```diff
@@ -58,9 +59,15 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
+    notEmptyData = async () => {
```

Comment:
It shouldn't happen (anymore), but just to ensure we don't pass on anything that would raise an error